README.md

```diff
@@ -26,12 +24,37 @@ In fact, we have a particular affection for food trucks. One of the great things
 
 Your assignment is to make it possible for us to find a food truck no matter where our work takes us in the city.
 
-This is a freeform assignment. You can write a web API that returns a set of food trucks (our team is fluent in JSON). You can write a web frontend that visualizes the nearby food trucks. We also spend a lot of time in the shell, so a CLI that gives us a couple of local options would be great. And don't be constrained by these ideas if you have a better one!
+Feel free to tackle this problem in a way that demonstrates your expertise of an area -- or takes you out of your comfort zone.
+
+## Technical Requirements
+
+### Interface
+
+You can write a simple REST service that returns a set of food trucks (our team is fluent in JSON).
 
-The only requirement for the assignment is that it give us at least 5 food trucks to choose from a particular latitude and longitude.
+### Expected Data Size
 
-Feel free to tackle this problem in a way that demonstrates your expertise of an area -- or takes you out of your comfort zone. For example, if you build Web APIs by day and want to build a frontend to the problem or a completely different language instead, by all means go for it - learning is a core competency in our group. Let us know this context in your solution's documentation.
+It would be great to open this solution up for other US cities so consider supporting large datasets. You should expect datasets in the millions.
```

Comment:
This line is unclear to me.  is it meant as "extra credit" or to have the candidate use this information in some way?  Are we trying to tell them to design their solution to handle millions of records?  Am I to go hunt for data from other cities to add to the solution  to "open this solution up"?  

If this sentence is meant to have the candidate take a specific action, I would make it more directly stated such that the candidate knows what to do.  Some possible examples: (if any of these actions are what is desired)
- Design the solution to handle millions of records
- Design the solution assuming that the dataset includes data from many cities with millions of records.
-  etc.